### PR TITLE
Added 2 more slides about Nepali Developers Survey

### PR DESCRIPTION
**Pull request by geshan**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -401,6 +401,25 @@
             <p class="fragment">A list of client libraries is available on <a href="http://oauth.net/2/">http://oauth.net/2/</a></p>
         </section>
         <section>
+            <h3>On a different node</h3>
```
Comment (ojhaujjwal) [inline index.html]: node or note?

File: index.html [hunk 2]
```diff
@@ -401,6 +401,25 @@
...
+            <ul>
+                <li class="fragment">Do you like <a href="http://bit.ly/1C6dACS">infographics</a> about Nepali software industry?</li>
+                <li class="fragment">Did you know, Laravel is the most popular framework in Nepal with <a href="http://bit.ly/sw-eng-np-pt1">40% usage</a></li>
+                <li class="fragment">Git is the most used VCS in Nepal with <a href="http://bit.ly/sw-eng-np-pt2">82.22 % usage</a></li>
+                <li class="fragment">22.22% developers in Nepal still <a href="http://bit.ly/sw-eng-np-pt3">deploy to production using FTP</a></li>
+            </ul>
+        </section>
+        <section>
+            <h3>On a different node</h3>
```
Comment (ojhaujjwal) [inline index.html]: same here

File: index.html (hunk 1 above)
Comment (geshan) [inline index.html]: will change.

File: index.html (hunk 2 above)
Comment (geshan) [inline index.html]: done.

Warm Regards,
Geshan Manandhar

Blog http://geshan.com.np | Twitter http://twitter.com/geshan

**Pull request merged**